runtime/pom.xml

```diff
@@ -25,7 +25,7 @@
         <annotations.api.version>6.0.53</annotations.api.version>
         <ehcahce.sizeOf.version>0.4.0</ehcahce.sizeOf.version>
 
-        <grpc.version>1.41.0</grpc.version>
+        <grpc.version>1.44.0</grpc.version>
```

Comment:
As discussed offline, hope you have checked my mvn dependency:tree that grpc version aligns with netty version and doesn't bring different tcnative version